routing/routing.go

```diff
@@ -214,7 +223,8 @@ loop:
 	if err != nil {
 		return nil, err
 	}
-	for _, iface := range ifaces {
+	for _, tmp := range ifaces {
+		iface := tmp
```

Comment:
NIT: Can we keep it as it is?

Reply:
This is a correct bug-fix as-is, since future code takes an address of iface.